packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -153,7 +153,9 @@ tbody.spectrum-Table-body {
    * the bottom border curved corners were cut off when using borders.
    */
 
-  /* Box shadow for bottom border for non-selected rows that aren't immediatly above a selected row. */
+  /* Box shadow for bottom border for non-selected rows that aren't immediatly above a selected row. Can't omit the bottom border for last row unlike listview
+   * due to how table rows always reserve 1px for the bottom border (results in a white gap on hover otherwise).
+   */
```

Comment:
Note for future reference since I forgot that I omitted this change on purpose. We can revisit if/when we refactor TableView rows not to be +1 px in height compared to the cells